src/models/types/Question.ts

```diff
@@ -0,0 +1,15 @@
+import {Document} from 'mongoose'
+import Subject from './Subject';
+import Answer from './Answer';
+
+export default interface Question extends Document {
+    content: string;
+    image: string;
+    subject: Subject;
+    correctAnswer: Answer;
+    answers: Answer[];
+    isDeleted: boolean;
+    accuracy: Enumerator;
```

Comment:
Enumerator ?

Reply:
You must create a enum for accuracy like that:
enum Accuracy{
    Absolute,
    Relatively
}